### DISCUSSION TITLE
Set light slider to 100% when lighthas no brightness controls

### DISCUSSION THREAD

**Discussion by poachedindividual** [Clooos/Bubble-Card#870] [Feature requests]
I use auto entities to auto populate all my lights as bubble sliders. 
Sometimes it can be hard to tell if a light is on when it is a light without a brightness setting, it is either on or off. 
Would be great if lights without brightness controls just auto set the slider to 100% 

e.g. Dining and Entry are both on: 
![image](https://github.com/user-attachments/assets/0bbaa4ae-9f48-47db-8aa3-789093823d5b)

**Comment by Clooos** [comment]
Hi! If they don't have brightness you can use the switch button type for these lights. That's what I do, but indeed I could change that.

**Comment by poachedindividual** [comment]
Yep, thanks ,I thought about that.   
  
Unfortunately the cards are auto populated on my dash by auto entities so they are all created as the same type which is slider.   
  

  

  
  
  
Thanks
    
Nick
  
  
  
  
  
  
>

**Reply by Clooos** [reply]
Makes sense, I could do something even better, like forcing the switch type if it's set to slider but that there is no brightness on it. I will try that.
